workers/prettier-worker.ts

```
@@ -45,31 +93,27 @@ let prettierLoadAttempted = false;
+		throw new Error(PRETTIER_NOT_LOADED_ERROR);
 	}
 
-	let config = await prettier.resolveConfig(request.filePath ?? "package.json", {
-		editorconfig: true,
-	});
-
-	// Override with provided config if available
-	if (request.configOverride) {
-		config = { ...config, ...request.configOverride };
-	}
+	const config = await resolveConfig(request.filePath, request.configOverride);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)